### PR TITLE
Update user space to 21. Add adaptive keys, better doc. Optionally using org. Lots  miscellaneous changes and small fixes.

### PR DESCRIPTION
**Pull request by EricGebhart**
## Description

A year of updates. working with qmk 21.  Org files now exist for most of the code.  The code generated from
org is mostly the same as before.

The org files can be ignored if desired.
Added adaptive keys extension and definitions as used in Hands Down.
Updated to work with the combo reference layers pushed in qmk 20.
Added new hands down base layers,  
Updated thumb clusters for kinesis and kyria and customized kyria.

Newer mcus and keyboards in consistent use.
kinesis with teensy, kyria with liatris, kyria with avr, customized kyria with liatris.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

Issues found in upgrading QMK, here are some.
*  My current configuration now works with QMK 21. 
*  Duplicate combo code is removed.
*  Renamed local keycodes.h to custom_keycodes.h to avoid name clash.

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/ericgebhart/config.h [hunk 1]
```diff
@@ -47,227 +66,177 @@
 
 #define TAPPING_TOGGLE  2
 #define TAPPING_TERM    200
-
+// Alans tapping term.
+// #define TAPPING_TERM 170 // Mod TAP VS HOLD timing in milliseconds
```
Comment (drashna) [inline users/ericgebhart/config.h:70]: This is also defined right above here?

**Comment by EricGebhart** [comment]
Yes. I wanted to remember that number. Seems silly to maintain a separate
document for that..
I guess I can remove the comment or change it to prose so it is not
commented code. Is there a preference? Seems more clear as is.

Le mer. 28 juin 2023, 11:37, Drashna Jaelre ***@***.***> a
écrit :

**Comment by drashna** [comment]
well, git keeps a history of the changes, so you can always review that.  And there are a number of tools that make viewing those changes easy, too.   So best is to just remove.

**Comment by EricaLinaG** [comment]
Sorry to take so long. I'm travelling the last month. Forgot my tablet and broke my computer the day after my last comment.
Finally got a working computer today.

**Comment by zvecr** [comment]
Unfortunately QMK has [changed its stance](https://docs.qmk.fm/#/ChangeLog/20230827?id=notable-changes) on accepting user keymaps to the repository.

> qmk/qmk_firmware is no longer accepting PRs for keymaps other than for manufacturer-supported keymaps. User keymap workflow has been documented [here](https://docs.qmk.fm/#/newbs) for several years. This change is to progressively reduce the maintenance burden on the project, and to allow us to focus on the core features of QMK.

QMK Firmware now officially supports [storing user keymaps outside of the normal QMK Firmware repository](https://docs.qmk.fm/#/newbs_external_userspace), allowing users to maintain their own keymaps without having to fork, modify, and maintain a copy of QMK Firmware themselves. 

While we appreciate the time that was taken on this PR, hopefully you understand we are closing it, so that QMK can improve in the long term.

**Pull request closed without merging**